src/Features/LanguageServer/Protocol/Handler/Completion/AbstractLspCompletionResultCreationService.cs

```diff
@@ -414,6 +414,19 @@ public int GetHashCode([DisallowNull] ImmutableArray<CharacterSetModificationRul
             return builder.ToArray();
         }
 
+        private async static Task<CompletionChange> GetCompletionChangeOrDisplayNameInCaseOfExceptionAsync(CompletionService completionService, Document document, CompletionItem completionItem, CancellationToken cancellationToken)
+        {
+            try
+            {
+                return await completionService.GetChangeAsync(document, completionItem, cancellationToken: cancellationToken).ConfigureAwait(false);
+            }
+            catch (Exception e) when (FatalError.ReportAndCatchUnlessCanceled(e))
+            {
+                // In case of exception, we simply return DisplayText with default span as the change.
```

Comment:
Does DisplayText have the risk of containing stars in the case of an IntelliCode suggestion?

Reply:
Ah, you are right. I thought I had moved them to use `DisplayTextPrefix`, turns out I never did.

Reply:
So is this going to insert stars into the file in that case? Is there a change to make here or should we just be moving to DisplayTextPrefix?

Reply:
We should take this change as is (i.e. fallback to using DisplayText), and I will make a separate change in Pythia to use DisplayTextPrefix for the star.